### DISCUSSION TITLE
update single container via http api mode

### DISCUSSION THREAD

**Discussion by aitchdot** [containrrr/watchtower#1731] [Q&A]
Is it possible to trigger an update with the `--http-api-update` flag, for a single container only but keep updating every container with the scheduled updates? I'm trying to implement the webhook call in to the corresponding containers gitlab-ci.yml

Thanks in Advance

**Comment by piksel** [comment]
Sorry for the late reply. There was an `images` query parameter added to the `/v1/update` API endpoint that should do what you need. So something like:
```
curl --oauth2-bearer TOKEN http://localhost:8080/v1/update?images=ORG/APP
```

**Reply by gersur** [reply]
https://containrrr.dev/watchtower/http-api-mode/

Not updated?

**Reply by piksel** [reply]
No, it's not in the docs for some reason...

**Reply by Huber1** [reply]
Found out that the parameter only works if in singular: `image` instead of `images`

**Comment by mrekin** [comment]
Hi!
Looking a way to update only container using http api.
If I'll set image name in api request - will watchtower update all containers based on this image?

**Reply by aitchdot** [reply]
That's indeed what i meant from the beginning

**Comment by Huber1** [comment]
Bit late to the party, but this works:
```bash
curl --oauth2-bearer TOKEN http://localhost:8080/v1/update?image=ORG/APP
```

nearly the same as @piksel mentioned, but the parameter needs to be `image` instead of `images`
